docker/lnd/Dockerfile

```diff
@@ -12,7 +12,17 @@ RUN curl -SLO https://github.com/lightningnetwork/lnd/releases/download/v${LND_V
   && tar -xzf *.tar.gz -C /opt \
   && rm *.tar.gz
 
+RUN curl -SLO https://github.com/lightningnetwork/lnd/releases/download/v${LND_VERSION}/lnd-linux-amd64-v${LND_VERSION}.tar.gz \
```

Comment:
Do we need to download  and extract the LND tar file again?

Reply:
copy & paste ;)